.github/workflows/interop-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Interoperability tests with GnuTLS and NSS
+on: [pull_request, push]
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      matrix:
+        COMPONENT: [openssl-upstream-gnutls, openssl-upstream-nss]
```

Comment:
We can do even more parallelization if desired, can get the overall time to under 30 minutes. For inspiration, GnuTLS config: https://gitlab.com/gnutls/gnutls/-/blob/master/.gitlab-ci.yml?ref_type=heads#L337

Reply:
Let's simplify this:
```
        COMPONENT: [gnutls, nss]
```